docs/modules/mapstore/pages/configuring-a-generic-maploader.adoc

```diff
@@ -4,13 +4,32 @@
 
 {description}
 
+NOTE: The objects created in the distributed map are stored in a GenericRecord. You can build a map interceptor to store the data in a POJO (Plain Old Java Object).
+
 For a list of all supported external systems, including databases, see available xref:external-data-stores:external-data-stores.adoc#connectors[data connection types].
 
-== Before you Begin
+== Before you begin
 
 You need a xref:external-data-stores:external-data-stores.adoc[data connection] that's configured on all cluster members.
 
-== Quickstart Configuration
+== Add dependencies
+
+If you are using a Hazelcast JAR file, you need to ensure you have added the following dependencies to your pom.xml:
```

Comment:
Better wording would be that they need to be on the classpath. E.g. if you use the full docker image they are. If you use the slim image the `hazelcast-mapstore` is missing.

Also the section `Supported backends` below also mentions that you need jdbc driver or in case of using MongoDB you need the `hazelcast-jet-mongodb` jar. Maybe this could be unified into single place.